src/io/io.jl

```diff
@@ -38,13 +38,7 @@ function load_restart_file!(dg::AbstractDg, restart_filename)
 
       # Read variable
       println("Reading variables_$v ($name)...")
-      if ndims(dg) == 2
-        dg.elements.u[v, :, :, :] = read(file["variables_$v"])
-      elseif ndims(dg) == 3
-        dg.elements.u[v, :, :, :, :] = read(file["variables_$v"])
-      else
-        error("Unsupported number of spatial dimensions: ", ndims(dg))
-      end
+      dg.elements.u[v, ..] = read(file["variables_$v"])
```

Comment:
Can you please change the access everywhere such that only the *nodes* access is hidden behind the ellipsis operator? This would be a workaround for ChrisRackauckas/EllipsisNotation.jl#20 (see also https://github.com/ChrisRackauckas/EllipsisNotation.jl/issues/20#issuecomment-695706237). I.e., like this:
```suggestion
      dg.elements.u[v, .., :] = read(file["variables_$v"])
```
This way the `..` only ever hides 3 dimensions and the performance impact is avoided. This also applies to all other places where you used it, of course.